docs/Makefile

```
@@ -18,3 +18,7 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+clean:
+	@$(SPHINXBUILD) -M clean "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	rm -rf $(SOURCEDIR)/getting_started/examples/*
```

```suggestion
	rm -rf "$(SOURCEDIR)/getting_started/examples/*"
```

Enclose these in quotes in case there is a whitespace in `SOURCEDIR`

---

Also is there a need for `-f` flag here?